src/ChainRulesCore.jl

```diff
@@ -19,13 +19,13 @@ export Tangent, NoTangent, InplaceableThunk, Thunk, ZeroTangent, AbstractZero, A
 include("compat.jl")
 include("debug_mode.jl")
 
-include("differentials/abstract_differential.jl")
-include("differentials/abstract_zero.jl")
-include("differentials/thunks.jl")
-include("differentials/composite.jl")
-include("differentials/notimplemented.jl")
+include("tangents/abstract_tangent.jl")
```

Comment:
should we call the folder `tangent_types` ?

Reply:
Yeah, I guess that makes sense